projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1427,6 +1427,11 @@ public function update_published_message( $messages ) {
 
 		$labels = array();
 		foreach ( $services as $service_name => $display_names ) {
+			// Twitter connections do not trigger Publicize anymore. Skip.
+			if ( 'Twitter' === $service_name ) {
```

Comment:
The capitalization caught my attention, but traced it back. It is `twitter` in the options table, but it is converted to ucfirst via https://github.com/Automattic/jetpack/blob/1cb88291aff63bf80754db0a8719759c4140c0fe/projects/packages/publicize/src/class-publicize-base.php#L347